docs/guides/serverless-hamsim-template.ipynb

```diff
@@ -0,0 +1,1033 @@
+{
```

Comment:
<h4>Exit early if using dry run mode</h4><p> </p><p>If dry run mode has been selected then the program is stopped before executing on hardware. This can be useful if, for example, you want first to inspect the two-qubit depth of the ISA circuit before deciding to execute on hardware.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/2276/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>